apple-pie/index.html

```diff
@@ -4,36 +4,53 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet"  href="/apple-pie/styles/style.css">
```

Comment:
```suggestion
    <link rel="stylesheet"  href="styles/style.css">
```
Si pones /apple-pie, le estas diciendo que dentro de la raiz tiene que haber una carpeta /apple-pie, pero de la raiz del ordenador! y eso no es del todo cierto, por ello es mejor quitarle eso (o si quieres poner la barra poner el ./), entonces por ello mejor quitarlo! mas facil jejee